src/schemas/entertainment.json

```diff
@@ -138,14 +149,7 @@
           "description": "Time when the entity is no longer entitled."
         },
         "offeringType": {
-          "type": "string",
-          "enum": [
-            "free",
-            "subscribe",
-            "buy",
-            "rent"
-          ],
-          "description": "The offering type of the WayToWatch."
+          "$ref": "https://meta.comcast.com/firebolt/entertainment#/definitions/OfferingType"
```

Comment:
This is totally a valid JSON $ref value, but could you please change it to be relative (it's referring to a definition in the same file), e.g. `#/definitions/OfferingType` w/out the protocol & domain?

Our tooling assumes that $refs that start with http are from other files, and this might generate strange docs if we do it this way.

Reply:
Actually, I guess we're doing this already elsewhere, and it works fine!